docs/08-docker/02-windows-docker-images.mdx

```diff
@@ -17,70 +17,94 @@ found in
 
 ### Microsoft does not allow redistribution of Visual Studio build tools
 
-Microsoft doesn't permit publishing images with VS Build tools
-[`discussion`](https://github.com/microsoft/Windows-Containers/issues/102#issuecomment-827170844) so
-aren't we allowed to do that. Building windows IL2CPP project in pure editor images leads to
-exceptions claiming that "C++ code builder" nor "Windows 10 SDK" are installed. There are two
-options to avoid this obstacle
+Microsoft does not permit publishing images with VS Build tools which are needed for IL2CPP builds 
+[`discussion`](https://github.com/microsoft/Windows-Containers/issues/102#issuecomment-827170844). 
+You will encounter exceptions in the Unity log similar to the below:
 
-1. Inject/mount Microsoft Visual Studio with VC++ and Windows 10 SDK from the host system into the
-   container during runtime.
-2. Installs vctools, e.g. `choco install visualstudio2017-workload-vctools --no-progress -y` and
-   push it to a private docker registry
+```
+error: Could not set up a toolchain for Architecture x64. Make sure you have the right build tools installed for il2cpp builds. Details:
+IL2CPP C++ code builder is unable to build C++ code. In order to build C++ code for Windows Desktop, you must have one of these installed:
+ * Visual Studio 2022 or newer with C++ compilers and Windows 10 SDK (recommended)
+ * Visual Studio 2019 with C++ compilers and Windows 10 SDK
+ * Visual Studio 2017 with C++ compilers and Windows 10 SDK
+ * Visual Studio 2015 with C++ compilers and Windows 10 SDK
+Visual Studio 2017 (or newer) is detected using `vswhere.exe` as well as VSCOMNTOOLS environment variables.
+Visual Studio 2015 is detected by looking at "SOFTWARE\Microsoft\VisualStudio\14.0_Config\InstallDir" in the registry as well as VSCOMNTOOLS environment variables.
+Windows 10 SDK is detected by looking at "SOFTWARE\Wow6432Node\Microsoft\Microsoft SDKs\Windows\v10.0\InstallationFolder" in the registry.
+Unable to detect any compatible Visual Studio installation!
+ * Found Visual Studio 2022 installation without C++ tool components
+Windows 10 SDK is not installed. You can install from here: https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk/
+```
```

Comment:
We're trying to keep the docs concise, so that it's in fact better read and understood by people. We also have a focus on making this beginner friendly. I feel putting the error in our documentation is overkill. 

Perhaps we can use a [collapsible](https://docusaurus.io/docs/next/markdown-features#details) component or leave it away altogether.

Reply:
Ah, yes I'm going to try the collapsible component route. I started looking for that when I dumped this in and then got distracted!